pynsot/app.py

```diff
@@ -53,7 +53,7 @@
     'devices': ['hostname'],
     'networks': ['cidr'],
     'attributes': ['name', 'resource_name'],
-    'interfaces': ['name', 'device'],
+    # 'interfaces': ['name', 'device'],
```

Comment:
For reference. The interfaces work will continue after this lands. That mapping will be used to ease to manage interfaces from the CLI without having to lookup `id` every time.